### DISCUSSION TITLE
Burndown chart?

### DISCUSSION THREAD

**Discussion by kilianmuster** [mgmeyers/obsidian-kanban#507] [Ideas & Feature Requests]
Would be great to have a way to show burndown charts or other graphs that help confirm whether you're on track to complete a huge list of tasks within a certain time frame or not.

**Comment by TheSkay** [comment]
Any news about this feature request ?

It can be very, very useful to have this feature.

**Comment by hittjwX** [comment]
If start and completion attributes were saved with each task, then it would be interesting to graph those. There would need to be a target date for completion outside the tasks to calculate burndown.
